docs/backward-compatibility.md

```diff
@@ -6,4 +6,5 @@ EarlGrey is supported on the following platforms:
 iOS Version        | Devices           | Simulators
 ------------       | ------------      | ------------
 iOS 8.x            | iPhone, iPad      | iPhone, iPad
-iOS 9.0 - 9.3      | iPhone, iPad      | iPhone, iPad
+iOS 9.x            | iPhone, iPad      | iPhone, iPad
+iOS 10 beta4       |         -         | iPhone, iPad
```

Comment:
didn't try device because i didn't have a spare phone to install 10b4 on. if you do, could you try?